### ISSUE TITLE
Translations with newlines are moved and replaced with empty string

### ISSUE THREAD

**Issue by bes** [Flinesoft/BartyCrouch#65]
Example

Command: `bartycrouch interfaces -p ./`

Several translations, all with newlines, are affected. The original has \n but the translations contain actual newlines, which BartyCrouch does not seem to like very much.

Example:
```
-/* Class = "UILabel"; text = "LIVE RECORDING\n00:00:00"; ObjectID = "y9M-C2-74f"; */
-"y9M-C2-74f.text" = "实时录制
-00:00:00";
-

******* some other translations ***********

+/* Class = "UILabel"; text = "LIVE RECORDING\n00:00:00"; ObjectID = "y9M-C2-74f"; */
+"y9M-C2-74f.text" = "";
```

So the problem is two-fold:

1. The translation goes from working to translation to empty (should not be touched by BartyCrouch)
2. Is relocated to another place in the file

How do I handle this properly?

**Comment by Dschee** [comment]
Thanks for this issue. Will have a look next week, when I have more time. Sorry for keeping you await.

**Comment by Dschee** [comment]
By default BartyCrouch adds translations with empty values to the end of your strings file. If you want to keep the base translation as the value in your languages, you need to specify the `--default-to-base` command (or the `-b` shorthand) – this will add new translations with their Storyboard content as value.

Other than that, in my tests all multiline translations (those including newlines) were handled correctly. Also reordering should only happen if you specify the [`--sort-by-keys` or `-s`](https://github.com/Flinesoft/BartyCrouch#sort-by-keys-aka--s---sort-by-keys-optional) option to the code subcommand. If it really happens to the interfaces command as well, it would be very helpful if you could add a test to the project and send a PR with it so we can fix the test in order to fix the issue.

**Comment by rebe1one** [comment]
I'm seeing something similar, it removes the translation completely for components with multiple lines of text. If I remove the newline it works as expected. 

On the other hand, I pulled down the latest BartyCrouch so I could write a test for this and it handles the newline case as expected, even for the file that I am having issues with.

**Comment by rebe1one** [comment]
Still looking into this, seems to be fine in the tests, but once I compile the bartycrouch binary and try to run it via command line it simply ignores strings that have newlines.

**Comment by rebe1one** [comment]
After looking into the regex, it seems like currently newlines are supported in the /* comment */ part of the string but not in the actual key-value pair line. I've opened a PR with a slightly modified regex that will pick up values with new lines.

**Comment by Dschee** [comment]
Hopefully fixed via #75.
